### PR TITLE
[I18N] account: propely translate Invoice PDF for all languages

### PR DESCRIPTION
**Pull request by lade-odoo**
Step to reproduce the issue:
1) Print an invoice linked to a company with any language != en_US (e.g.: es).

Solution: Following commit [1], the alignement of the concerned lines in the
XML has been changed (adding one `\t`). Consequently, the translations could
not be applied as the ID had changed.

[1]: https://github.com/odoo/odoo/commit/72379768080cfd05666e5b7bfbea6a976b2ea263

opw-2860541

### CODE REVIEW

**Review by nle-odoo — APPROVED** [review]
Hi,

> Change was not done in Transifex because the change affected all the languages and it was easier to do it locally.

the change you are doing in PO files would be usually overridden on the Sunday after this is merged (with weekly transifex to Odoo source update that will override any change in a *.PO that is not in transifex).

The last synchronization of 15.2 seems to be from 9 days ago, so it seems that changing *.PO could work if 15.2 is no longer updated.

For 15.3 the last synchronization is from 2 days ago so it seems to be the one that is being currently updated, so for 15.3 the PO change should probably not be done (and people need to fix it in transifex once the POT change is merged).

**Comment by mart-e** [comment]
> The last synchronization of 15.2 seems to be from 9 days ago, so it seems that changing *.PO could work if 15.2 is no longer updated.

Indeed, we have just made the switch. The transifex project transifex.com/odoo/odoo-master/ now contains the saas-15.3 content.
If you need to change the saas-15.2 translations, it should be done on github like this.
If you need to change the saas-15.3 source, you should change only for the languages _not_ on https://www.transifex.com/odoo/odoo-master/languages/

**Comment by MiniLau** [comment]
@fw-bot ignore

**Comment by MiniLau** [comment]
We can keep the PR for changes in >= 15.3 for languages that are not present in Transifex. We also need the change in the .dot file to create the entry in transifex (automatically as we are in community).
@fw-bot up to master

**Comment by MiniLau** [comment]
@fw-bot up to saas-15.3
Easier to stop the fw in 15.3, make the appropriate change there and start again the fw up to master

**Comment by nle-odoo** [comment]
> Easier to stop the fw in 15.3, make the appropriate change there and start again the fw up to master

I don't think you can change target after this PR is merged. But if you make a push in 15.3 forward-port (and don't merge 15.4 and master), the change should automatically be pushed on 15.4 and master.

fw-bot up to master

robodoo r+

**Pull request closed without merging**
